lib/logstash/outputs/opensearch/http_client/manticore_adapter.rb

```diff
@@ -76,13 +79,34 @@ def aws_iam_auth_initialization(options)
       @credentials = Aws::CredentialProviderChain.new(credential_config).resolve
     end
 
+    def basic_auth_initialization(options)
+      set_user(options)
```

Comment:
why not create single method to set both user name and password?

Reply:
yeah, we can do that. I just thought of defining separate setter methods to make sure everything looks uniform.

Reply:
updated